src/app/pages/BuyPage/components/Button/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import styled, { css } from 'styled-components/macro';
+
+interface BtnProps {
+  onClick: () => void;
+  disabled?: boolean;
+}
+
+interface Props extends BtnProps {
+  text: React.ReactNode;
+}
+
+const StyledButton = styled.button`
```

Comment:
Pls update the styles using `module css` instead of `styled-compoments`.

Reply:
fixed

Reply:
it still uses styled component